samples/bluetooth/peripheral_ots/src/main.c

```diff
@@ -168,16 +171,17 @@ static int ots_init(void)
 	}
 
 	/* Prepare first object demo data and add it to the instance. */
-	for (uint32_t i = 0; i < sizeof(objects[0]); i++) {
-		objects[0][i] = i + 1;
+	for (uint32_t i = 0; i < sizeof(objects[0].data); i++) {
+		objects[0].data[i] = i + 1;
 	}
 
 	memset(&obj_init, 0, sizeof(obj_init));
-	obj_init.name = "first_object.txt";
+	strcpy(objects[0].name, "first_object.txt");
```

Comment:
Since `CONFIG_BT_OTS_OBJ_MAX_NAME_LEN` may be lower than `strlen("first_object.txt")` should we add some check here?

Reply:
I guess the same check should be added for the Directory Listing Object's name. The default value of `CONFIG_BT_OTS_OBJ_MAX_NAME_LEN` is large enough to hold the default value of `CONFIG_BT_OTS_DIR_LIST_OBJ_NAME` but a user could misconfigure these two values such that the  `CONFIG_BT_OTS_DIR_LIST_OBJ_NAME` is too large.